File: Server/ElectricityBillPayment/StudentPortal.DTO/DTO/UserRole.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace ModelClass.DTO
{
    public class UserRole
    {
        public UserRole(){
            }
        public int UserRoleId { get; set; }
        public int RoleId { get; set; }
        public int UserId { get; set; }
        public string CreatedBy { get; set; }
        public DateTime CreatedDate { get; set; }
        public string UpdatedBy { get; set; }
        public DateTime? UpdatedDate { get; set; }
        public int Status { get; set; }
        public  Role Role { get; set; }
        public  User User { get; set; }
    }
}

File: Client/ElectricityBillClient/src/app/Model/UserRole.ts
import { Role } from './Role';
import { User } from './User';

export class UserRole  {
           UserRoleId:number=0;
           RoleId:number=0;
           UserId:number=0;
           CreatedBy:string='';
           CreatedDate:Date|undefined;
           UpdatedBy:string='';
           UpdatedDate:Date|undefined;
           Status:number=0;
           Role:Role = new Role;
           User:User = new User;
}
